lib/api-client.js

```diff
@@ -5,6 +5,10 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
   'Content-Type': 'application/json',
   'Accept': 'application/vnd.api+json; version=1',
 }, {
+  params: {
+    http_cache: true
```

Comment:
OK, `window.location.search` now overrides the defaults, so `?http_cache=false` in the window will be passed through to the client.